doc/en/changelog.rst

```diff
@@ -28,6 +28,13 @@ with advance notice in the **Deprecations** section of releases.
 
 .. towncrier release notes start
 
+pytest 6.2.4 (2021-05-04)
```

Comment:
```suggestion
pytest 6.2.4 (2021-05-04)
=========================

```

Weird, did towncrier failed here?

(Anyway I say let's move on with the release and investigate this later).